### ISSUE TITLE
Copy a specific user

### ISSUE THREAD

**Issue by victalejo** [juliansantosinfo/iqoption-bot-openbot#1]
If you can copy the operations of the ranking, can you copy the operations of a specific user? Thank you very much. You have made an excellent script.

**Comment by juliansantosinfo** [comment]
Yes, just change/replace the "filterRanking" function to append the id of specific users in the "filter" array.
